src/steps/test-projects.ts

```diff
@@ -23,16 +23,30 @@ export default async function testProjects(testProjectPaths: string[], finalResu
 				execa('yarn', [`test`], {
 					cwd: testProjectPath,
 					all: true,
-				}).catch((error) => {
```

Comment:
I opted to handle the rejection as the second argument to `.then`, instead of increasing the the number of chained methods with the same call backs we are using now, I could revert and use the `.then`, though.

Reply:
I think chaining `.then()` with `.catch()` will be more readable than what you have right now.